lambdas/transcode/index.py

```diff
@@ -3,18 +3,30 @@
 """
 import subprocess
 import tempfile
+from urllib.parse import urlparse
 
 from t4_lambda_shared.decorator import api, validate
 from t4_lambda_shared.utils import get_default_origins, make_json_response
 
+# Map of supported content types and corresponding FFMPEG formats
+FORMATS = {
+    'video/mp4': 'mp4',
+    'video/webm': 'webm',
+    'audio/mpeg': 'mp3',
+    'audio/ogg': 'ogg',
+}
+
+# TODO: Remove this.
+LEGACY_FORMATS = {v: k for k, v in FORMATS.items()}
+
 SCHEMA = {
     'type': 'object',
     'properties': {
         'url': {
             'type': 'string'
         },
         'format': {
-            'enum': ['mp4', 'webm']
+            'enum': list(FORMATS) + list(LEGACY_FORMATS)
```

Comment:
not a bug as written but it could become a bug if keys and values overlap so might wanna check that e.g. with `len(set(SCHEMA['format']['enum']))`

Reply:
I'd just remove the legacy formats as soon as nl0 updates his code